driver/ppm_events_public.h

```diff
@@ -1283,6 +1303,7 @@ struct ppm_evt_hdr {
 
 extern const struct ppm_name_value socket_families[];
 extern const struct ppm_name_value file_flags[];
+extern const struct ppm_name_value file_modes[];
```

Comment:
Where is this coming from? You could have a look at how/where `file_flags`, `flock_flags` and similar are defined to implement this.